docs/dev/create_a_tile.md

```diff
@@ -0,0 +1,321 @@
+| Color Mode         | RGB                                                                                                                                               |
+| Color Profile      | sRGB                                                                                                                                              |
+| Description        | The maximum number of characters allowed is 300.                                                                                                  |
+
+Follow this template to define the `media` object in the `manifest.json` file which includes an image, a video thumbnail, and a video:
+
+{{< code-block lang="json" filename="manifest.json" collapsible="true" >}}
+"media": [
+      {
+        "media_type": "image",
+        "caption": "A Datadog Marketplace Integration OOTB Dashboard",
+        "image_url": "images/integration_name_image_name.png"
+      },
+      {
+        "media_type": "video",
+        "caption": "A Datadog Marketplace Integration Overview Video",
+        "image_url": "images/integration_name_video_thumbnail.png",
+        "vimeo_id": 123456789
+      },
+    ],
+{{< /code-block >}}
+
+For more information, see [Integrations Assets Reference][22].
+
+## Open a pull request
+
+Before you open a pull request, run the following command to catch any problems with your integration:
+
+```
+ddev validate all <INTEGRATION_NAME>
+```
+
+Complete the following steps:
+
+1. Commit all changes to your feature branch.
+2. Push your changes to the remote repository. 
+3. Open a pull request that contains your integration tile's asset files (including images) in the [`marketplace`][18] or [`integrations-extras`][26] repository. 
+
+After you've created your pull request, automatic checks run to verify that your pull request is in good shape and contains all the required content to be updated.
+
+## Review process
+
+Once your pull request passes all checks, reviewers from the `Datadog/agent-integrations`, `Datadog/marketplace-review`, and `Datadog/documentation` teams provide suggestions and feedback on best practices.
+
+Once you have addressed the feedback and re-requested reviews, these reviewers approve your pull request. Contact the Marketplace team if you would like to preview the tile in your sandbox account. This allows you to validate and preview your tile before your tile is live to all customers.
+
+## Troubleshoot errors
+
+Out-of-the-box integrations in the `integrations-extras` repository can run into validation errors when the forked repository is out of date with the origin. 
+
+To resolve validation errors, update the forked repository on the GitHub web app: 
+
+1. In [GitHub][29], navigate to your forked `integrations-extras` repository.
+1. Click **Sync fork** under the green **Code** dropdown menu and click **Update branch**.
```

Comment:
```suggestion
1. In [GitHub][29], navigate to your forked `integrations-extras` repository.
1. Click **Sync fork** and click **Update branch**.
```
Rephrased cos this might make the user think the button is inside the drop-down menu. A screenshot might be helpful here.